circle.yml

```diff
@@ -12,6 +12,6 @@ test:
     - make test
     - nvm install 0.12.7
     - nvm alias default 0.12.7
```

Comment:
@svozza, could we `nvm use 0.12.7 && ...` here instead, or is there a reason this is different from the lines below?